src/app/login/state/auth.actions.ts

```diff
@@ -0,0 +1,18 @@
+import { createAction, props } from '@ngrx/store';
+import { User } from '../shared/models/user.model';
+
+const AUTHENTICATE = 'AUTHENTICATE';
```

Comment:
for actions type quite often are used Enums https://www.typescriptlang.org/docs/handbook/enums.html
for example 
export enum CourseActionType {
  ADD_ITEM = '[COURSE] Add Course',
  REMOVE_ITEM = '[COURSE] Remove Course',
}